### ISSUE TITLE
Timeout on GitHub Actions

### ISSUE THREAD

**Issue by ardaerzin** [zenclabs/viteshot#66]
switching from react-screenshot-test, everything works fine locally but we keep getting a timeout error while running the tests on GitHub Actions. logs look like the following:

```
Pre-bundling dependencies:
  react
  clsx
  next/image
  styled-components
  styled-css-grid
  (...and 17 more)
(this will be run only when your dependencies or config have changed)

warn - You have enabled the JIT engine which is currently in preview.
warn - Preview features are not covered by semver, may introduce breaking changes, and can change at any time.
TimeoutError: **Navigation timeout of 30000 ms exceeded**
error Command failed with exit code 1.
```

I did a debug locally, and like I said, I don't run into any problems there

**Comment by fwouts** [comment]
Hi @ardaerzin, sorry for the delay!

I wonder if you can reproduce it locally if you delete the `node_modules/.vite` directory?

**Comment by ardaerzin** [comment]
@fwouts no worries. 

I just tried your suggestion, multiple times, but was not able to reproduce it.

**Comment by ardaerzin** [comment]
@fwouts we were using Percy before, so to keep things consistent, I was trying the Percy shooter. today I gave the playwright shooter a go, and I do not experience this problem

**Comment by fwouts** [comment]
Thanks @ardaerzin, so just to confirm, the issue was specific to the Percy shooter?

**Comment by ardaerzin** [comment]
@fwouts yes that seems to be the case. having no issues with the playwright shooter now. if it happens again, I'll update this ticket with further info
